app/ui/src/app/integration/edit-page/step-configure/filter-steps/basic-filter.component.ts

```diff
@@ -103,8 +103,7 @@ export class BasicFilterComponent implements OnChanges {
     this.integrationSupport
       .getFilterOptions(dataShape)
       .toPromise()
-      .then((resp: any) => {
-        const body = JSON.parse(resp['_body']);
+      .then((body: any) => {
```

Comment:
Keep in mind you won't want to cherry pick this part onto the 1.2.x branch.

Reply:
Right need to fix that, it’s in there now